include/bpf.h

```diff
@@ -77,6 +77,20 @@ bpf_link_get_next_id(__u32 start_id, __u32* next_id);
 int
 bpf_create_map(enum bpf_map_type map_type, int key_size, int value_size, int max_entries, __u32 map_flags);
 
+/**
+ * @brief Create a new map.
+ *
+ * @param[in] create_attr Structure of attributes to create a map using.
```

Comment:
nit: Structure of attributes using which a map gets created.

Reply:
Done